webviews/src/styles.css

```
@@ -182,12 +186,275 @@ body {
+/* Branch base picker: heavier primary toolbar control + command-palette popover. */
+#base-picker {
+  position: relative;
+  display: flex;
+  align-items: center;
+  min-width: 0;
+  /* Base is the primary action of this toolbar: it claims the remaining space in
+     toolbar-left and holds a floor so the compact, highly-shrinkable source/repo
+     selects are the crush victims, not the base button. The min-width keeps at
+     least `Base: <ref>` visible at realistic diff-panel widths. */
+  flex: 1 1 auto;
+  min-width: 156px;
```

Biome (2.5.0)

[error] Duplicate properties can lead to unexpected behavior and may override previous declarations unintentionally.

(lint/suspicious/noDuplicateProperties)